api.raml

```diff
@@ -633,3 +703,25 @@ traits:
         Configuration files can be only updated by users who has edit access on the project or the secret. Encrypted environment variables can't be edited.
       is: [withRequestItem: {item : ConfigFilePatch},
           withResponseItem: {item : ConfigFile}, withNotFoundError]
+
+/pipelines:
+  (stability): "prototype"
+  displayName: Pipelines
+
+  get:
+    displayName: List pipelines
+    is: [withResponseItems: {item : Pipeline}, withNotFoundError]
+  post:
+    displayName: Schedule pipeline execution
+    is: [withRequestItem: {item : PipelinePost},
+        withCreateResponse: {item : Pipeline}, withNotFoundError]
+
+  /{ppl_id}:
```

Comment:
This should be `pipeline_id` if we want to stay consistent with the rest of the documentation.